### PR TITLE
Add Multi-Task Explainable Recommendation (MTER) Model

### PR DESCRIPTION
**Pull request by lthoang**
### Checklist:

- [ ] I have added tests.
- [x] I have updated the documentation accordingly.
- [x] I have updated `README.md` (if you are adding a new model).
- [x] I have updated `examples/README.md` (if you are adding a new example).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -139,6 +139,7 @@ The recommender models supported by Cornac are listed below. Why don't you join
 | 2018 | [Collaborative Context Poisson Factorization (C2PF)](cornac/models/c2pf), [paper](https://www.ijcai.org/proceedings/2018/0370.pdf) | N/A | [c2pf_exp.py](examples/c2pf_example.py)
 |      | [Probabilistic Collaborative Representation Learning (PCRL)](cornac/models/pcrl), [paper](http://www.hadylauw.com/publications/uai18.pdf) | [requirements.txt](cornac/models/pcrl/requirements.txt) | [pcrl_exp.py](examples/pcrl_example.py)
 |      | [Variational Autoencoder for Collaborative Filtering (VAECF)](cornac/models/vaecf), [paper](https://arxiv.org/pdf/1802.05814.pdf) | [requirements.txt](cornac/models/vaecf/requirements.txt) |
+|      | [Multi-Task Explainable Recommendation (MTER)](cornac/models/mter), [paper](https://arxiv.org/pdf/1806.03568.pdf) | [requirements.txt](cornac/models/mter/requirements.txt) | [c2pf_exp.py](examples/mter_example.py)
```
Comment (tqtg) [inline README.md]: The models are listed alphabetically in the year. Thus, it should be above VAECF.

File: docs/source/models.rst [hunk 2]
```diff
@@ -20,6 +20,11 @@ VAE for Collaborative Filtering (VAECF)
 .. automodule:: cornac.models.vaecf.recom_vaecf
    :members:
 
+Multi-Task Explainable Recommendation (MTER)
```
Comment (tqtg) [inline docs/source/models.rst]: Same as the previous comment.

File: README.md (hunk 1 above)
Comment (lthoang) [reply README.md]: Do you mean above PCRL?

Comment (tqtg) [reply README.md]: Oh yes :)

**Pull request merged**